sdcm/sct_config.py

```diff
@@ -1927,6 +1931,8 @@ def __init__(self):
         if self.get("k8s_enable_sni") and not self.get("k8s_enable_tls"):
             raise ValueError("'k8s_enable_sni=true' requires 'k8s_enable_tls' also to be 'true'.")
 
+        SCTCapacityReservation.get_cr_from_aws(self)
```

Comment:
Shouldn't we raise an error if this new option is set to `true` and the cluster backend is not AWS?

Reply:
Like any other backend specific configuration, we should make sure we ignore it

For example we don't fail if some k8s settings are set, or when having a case that specific aws instance types while running on GCP.

We should add validation only if it's common gotcha, not something that would force us to split to multiple configuration or multiple pipelines

Reply:
K8S settings have the "k8s" prefix or "gke" or "eks".
This one doesn't. If it is expected to be applied only to aws then need to add appropriate prefix.

Reply:
still we have aws/gce/etc specific options that don't fail, e.g. `data_volume_disk_type` or `use_placement_groups`

Reply:
the a cosmetic issue, regarding configuration name prefixes.
one can later search and replace those.

the logic is that we don't introduce validation just for the sake of it.

I don't think this case needs one.